src/app/wallets/ln-send-payment.ts

```diff
@@ -389,6 +377,23 @@ const executePaymentViaIntraledger = async ({
     const updatedWalletInvoice = await invoicesRepo.update(walletInvoice)
     if (updatedWalletInvoice instanceof Error) return updatedWalletInvoice
 
+    const recipientWalletBalance = await getBalanceForWallet({
```

Comment:
Agreed. Instead of including the balance in every update type here, how about we let the user query the me field within that subscription? They can get all wallets balances and new transactions data to update any that's cached.

Here's what I came up for that: https://github.com/GaloyMoney/galoy/pull/745/commits/e7441b3eb395d7996bc489858371046956942b38